Backpack/Spinner/Classes/BPKSpinner.m

```diff
@@ -115,6 +115,8 @@ + (UIColor *)colorForSpinnerStyle:(BPKSpinnerStyle)style {
     switch (style) {
     case BPKSpinnerStylePrimary:
         return [BPKColor dynamicColorWithLightVariant:BPKColor.skyBlue darkVariant:BPKColor.white];
+    case BPKSpinnerStyleSecondary:
+        return BPKColor.textPrimaryColor;
     case BPKSpinnerStyleDark:
```

Comment:
Should we remove `BPKSpinnerStyleDark` and `BPKSpinnerStyleLight`?

Reply:
That would make the change breaking - and I think they're possibly still useful (especially light, because there could be UI that always has a dark background)